test/SPIRV/llvm.memmove.ll

```diff
@@ -0,0 +1,50 @@
+; RUN: llvm-as %s -o %t.bc
```

Comment:
can we make it a transcodeing test to make sure the generated SPIRV can be translated back to LLVM IR?